keepassxc-browser/background/keepass.js

```diff
@@ -720,6 +782,19 @@ keepass.setCryptoKey = function(id, key) {
     keepass.saveKey(keepass.databaseHash, id, key);
 };
 
+keepass.getAllKeys = function() {
```

Comment:
minor: recommend rename to simply `getKeys`. If you want to get a specific key, then the function would be `getKey(attribute)`

Reply:
I'd like to keep this implementation intact. The function will have some changes anyway when I'm finished with the improvements for unlocked/locked databases detection.